CHANGELOG.md

```diff
@@ -34,7 +34,7 @@
 - Fix `ContactPatientBottomSheet` not going back to call patient view on back click in call later mode
 - Fix overdue list not changing when switching facility from overdue screen
   
-## On Demo
+## 2021-07-13-7862
```

Comment:
It looks like we haven't moved the hotfixed changelog into the demo section. Can we do that as well, please?